codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/customization/AwsCustomGoDependency.java

```diff
@@ -45,7 +45,7 @@ private AwsCustomGoDependency() {
     }
 
     private static final class Versions {
-        private static final String INTERNAL_S3SHARED = "v0.1.0";
+        private static final String INTERNAL_S3SHARED = "v0.0.0-20201014161145-592e24615bd4";
```

Comment:
nit probably should be using the pseudo version `v0.1.1-0.`... Those seem to be the most compatible with existing modules.